prometheus/doc/alert-manager.md

```diff
@@ -0,0 +1,34 @@
+# Configuration
```

Comment:
>Configuration [](start = 2, length = 13)

please work with Yanjie to include a entry to this md from [https://github.com/Microsoft/pai/blob/master/pai-management/doc/how-to-write-pai-service-configuration.md](https://github.com/Microsoft/pai/blob/master/pai-management/doc/how-to-write-pai-service-configuration.md).

Reply:
ok, will cherry-pick to master after this PR is merged.